assets/css/main.css

```diff
@@ -4775,7 +4775,10 @@ input.field--reset:active {
 
 .tooltip-closeBtn {
   float: right;
-  padding: 4px; }
+  padding: 5px !important; }
```

Comment:
is there a way to escape the `!important` part?

Reply:
@v-gtsolov The button icon prop overwriting padding of the icon

Reply:
can we make stronger selector for it than, and remove the `!important` part

Reply:
This will get revisited by us with design and may be reverted back to 20px, however this is good for now